lib/general.js

```diff
@@ -0,0 +1,47 @@
+import { ok, nok } from './utils';
+import { exec } from 'teen_process';
+import { DoctorCheck } from './doctor';
+import NodeDetector from './node-detector';
+
+
+let checks = [];
+
+// Node Binary
+class NodeBinaryCheck extends DoctorCheck {
```

Comment:
seems odd to me that someone would be able to run this without the `node` binary being present. but maybe this is somehow possible given differing environments?